config/config.env

```diff
@@ -0,0 +1,2 @@
+GOOGLE_CLIENT_ID = 242608651265-l7rmod0inedei6t1sjpob8h0fruul5vp.apps.googleusercontent.com
```

Comment:
Do you need these? server/config/passport.js reference the process.env file which holds the IDs